### PR TITLE
Welcome Screen updates + theme fixes

### PR DESCRIPTION
**Pull request by crocsandcoffee**
## 📑 What does this PR do?

+ UX updated to the Welcome Screen to get us started
+ Updated Desktop app root to Welcome Screen
+ Theme fixes
+ Added placeholder app icon. (Pulled from Icons8)

# ✅ Checklist

- [ ] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] All the tests have passed

## 🧪 How can this PR been tested?
Run the android and desktop app

## 🧾 Tasks Remaining: (List of tasks remaining to be implemented)
- Hook up the SignUp and LoginIn button to new screens
- Figure out how we can share an app icon image between platform modules

- What is remaining to be implemented in this PR? Mention a list of them

Screen Shot 2022-11-12 at 1 27 03 PM

### CODE REVIEW

File: ui/welcome/src/commonMain/kotlin/social/androiddev/welcome/composables/WelcomeScreen.kt [hunk 1]
```diff
@@ -1,50 +1,136 @@
...
+import social.androiddev.common.utils.loadImageIntoPainter
 
 @Composable
 fun WelcomeScreen(
     modifier: Modifier = Modifier,
+    navigateToLogin: () -> Unit,
+    navigateToSignUp: () -> Unit,
+    appIcon: @Composable () -> Unit = {
+        AsyncImage(
+            load = { loadImageIntoPainter(url = "https://picsum.photos/200") },
```
Comment (thebino) [inline ui/welcome/src/commonMain/kotlin/social/androiddev/welcome/composables/WelcomeScreen.kt:42]: Is it fine to have this image here? license?
If not, switch to a placeholder: https://via.placeholder.com/200x200/6FA4DE/010041?text=MastodonX

Comment (crocsandcoffee) [reply ui/welcome/src/commonMain/kotlin/social/androiddev/welcome/composables/WelcomeScreen.kt:42]: Yes - open source random image generator. It's not static. And will be replaced shortly with a real app asset/image

**Review by thebino — APPROVED** [review]
If the `ic_elephant` and the placeholder image url do not pose a risk regarding their license, I'm fine

**Comment by crocsandcoffee** [comment]
> If the `ic_elephant` and the placeholder image url do not pose a risk regarding their license, I'm fine

It's pulled from icons8. 
https://icons8.com/license

But just to be safe I'll replace it with a built in icon from material before I merge. Same with the image url.

**Pull request merged**